test/tests/kernels/time_integration/tests

```diff
@@ -12,7 +12,7 @@
     type = 'Exodiff'
     input = 'hht.i'
     exodiff = 'hht_out.e'
-    verification = 'V1-02.md'
+    verification = 'hht.md'
```

Comment:
also here. The comparison done here is actually verification, not validation, as described in the .md file. I know you didnt write it, but it would be helpful if you could just change it in this PR.